tests/schema/tests.py

```diff
@@ -532,6 +532,23 @@ def test_alter_text_field(self):
         with connection.schema_editor() as editor:
             editor.alter_field(Note, old_field, new_field, strict=True)
 
+    @skipUnlessDBFeature('can_defer_constraint_checks', 'can_rollback_ddl')
+    def test_alter_fk_checks_deferred_constraints(self):
+        """
+        #25492 - Altering a foreign key's structure and data in the same
+        transaction should work on backends supporting it.
+        """
```

Comment:
It's helpful. The only question is about what style we should use in general (https://code.djangoproject.com/ticket/27392#comment:9). "Altering a foreign key's structure and data in the same transaction" might be sufficient rather than "should work" in most every docstring in this file. Your opinion is welcome on that.